wear/src/main/java/com/example/wear/snippets/tile/Tile.kt

```diff
@@ -200,3 +207,42 @@ class DynamicHeartRate : TileService() {
         )
     // [END android_wear_tile_dynamic_heart_rate]
 }
+
+class FeatureFallback : TileService() {
+    override fun onTileRequest(requestParams: RequestBuilders.TileRequest): ListenableFuture<Tile> {
+
+        // [START android_wear_tile_version_fallback]
+        val rendererVersion = requestParams.deviceConfiguration.rendererSchemaVersion
+
+        val arcElement =
+            if (
+                rendererVersion.major > 1 ||
+                (rendererVersion.major == 1 && rendererVersion.minor >= 500)
```

Comment:
What's the significance of the 1 and 500 - can these be extracted to descriptive constants?